### PR TITLE
Vasp constrained dynamics

### PR DESCRIPTION
**Pull request by sudarsan-surendralal**
A separate module derived from `Vasp` is created to run constrained MD, slow growth MD, and metadynamics in VASP.

### CODE REVIEW

File: pyiron/base/generic/parameters.py [hunk 1]
```diff
@@ -980,3 +980,11 @@ def _find_line(self, key_name):
                                                               self._dataset["Value"][i]))
             error_msg = "\n".join(error_msg)
             raise ValueError(error_msg)
+
+    def clear_all(self):
```
Comment (jan-janssen) [inline pyiron/base/generic/parameters.py:984]: Can not you use `load_default()` ?

Comment (sudarsan-surendralal) [reply pyiron/base/generic/parameters.py:984]: In this case yes. But what if `load_default()` does not give an empty dataset. In that case, wouldn't `clear_all` be useful?

Comment (jan-janssen) [reply pyiron/base/generic/parameters.py:984]: You are right.

**Comment by jan-janssen** [comment]
@sudarsan-surendralal you can merge it now.

**Comment by sudarsan-surendralal** [comment]
Cool! Let me quickly add a little testing!

**Pull request merged**